src/components/CompareResults/ResultsTable.tsx

```diff
@@ -104,9 +104,17 @@ export default function ResultsTable() {
   const [searchTerm, setSearchTerm] = useState(initialSearchTerm);
   const [frameworkIdVal, setFrameworkIdVal] = useState(frameworkId);
   const [tableFilters, setTableFilters] = useState(
-    new Map() as Map<string, Set<string>>, // ColumnID -> Set<Values to remove>
+    new Map([['confidence', new Set(['High'])]]), // ColumnID -> Set<Values to remove>// Default confidence filter
```

Comment:
Have you tried the code locally? This does the opposite of what you think: this _removes_ all items with the "high" confidence, we want to do the opposite.